web-module/src/main/resources/templates/add.html

```diff
@@ -48,12 +48,12 @@ <h3>Add User</h3>
         <div class="form-group">
             <div class="col-md-4">
                 <td>
-                    <select class="form-control mb-4" th:name="role" required>
+                    <select class="form-control mb-4" th:name="roleDTO.id" required>
                         <option value="">Choose User Role</option>
-                        <option value="ADMINISTRATOR">ADMINISTRATOR</option>
-                        <option value="SALE">SALE</option>
-                        <option value="CUSTOMER">CUSTOMER</option>
-                        <option value="SECURE API">SECURE API</option>
+                        <option value="1">ADMINISTRATOR</option>
```

Comment:
Bad practice to hardcode id. Need to get from list of roles

Reply:
done